### PR TITLE
Add evaluate.py

### PR DESCRIPTION
**Pull request by yuyu2172**
Merge after #9, #10

### CODE REVIEW

**Comment by knorth55** [comment]
Thank you for your contribution!
I was working on my repo today, and I made a big change.
Sorry for that, but can you rebase master branch?
Main change is
- format output order:  `bbox, mask, label, score`
- `predict` and `prepare` receives `(C, H, W)` image as input
- `coco_label_names` includes `__background__` as `0`
- dataset returns `BGR` image

File: examples/coco/evaluate.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        pred_masks.append(outputs[0][0])
+        pred_bboxes.append(outputs[1][0])
+        pred_labels.append(outputs[2][0] - 1)
+        pred_scores.append(outputs[3][0])
+
+    results = eval_instance_segmentation_coco(
+        sizes, pred_masks, pred_bboxes, pred_labels, pred_scores,
+        gt_masks, gt_bboxes, gt_labels, gt_crowdeds, gt_areas)
+
+    keys = ['ap/iou=0.50:0.95/area=all/maxDets=100',
```
Comment (knorth55) [inline examples/coco/evaluate.py]: I prefer this format for this kind of list.
can you format as below?

```
keys = [
    'ap/iou=0.50:0.95/area=all/maxDets=100',
]
```

**Comment by yuyu2172** [comment]
OK. I reflected your changes.

File: examples/coco/evaluate.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+from fcis.evaluations.eval_instance_segmentation_coco import \
+    eval_instance_segmentation_coco
+
+
+filepath = osp.abspath(osp.dirname(__file__))
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--data_dir')
```
Comment (knorth55) [inline examples/coco/evaluate.py]: `--data_dir` -> `--data-dir`

File: examples/coco/evaluate.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
```
Comment (knorth55) [inline examples/coco/evaluate.py]: can you make this file as executable file as `chmod +x evaluate.py`?

**Pull request merged**